rebar.lock

```diff
@@ -5,7 +5,7 @@
   0},
  {<<"aecuckoo">>,
   {git,"https://github.com/aeternity/aecuckoo.git",
-      {ref,"fa3d13e8c7003589153223f634c851d389b61b93"}},
+      {ref,"a1bff574dd0a965d3f9f2e23f980bbead8b4638d"}},
```

Comment:
https://github.com/aeternity/aecuckoocompare/fa3d13e8c7003589153223f634c851d389b61b93...a1bff574dd0a965d3f9f2e23f980bbead8b4638d

Reply:
https://github.com/aeternity/cuckoo/compare/3ae6195a67d9dcc33b1d82fd38f9bb82f2d29ee8...3fbc70bab20859e19dbc2341529ad44b4e7a0cef

Reply:
Main change is https://github.com/aeternity/cuckoo/pull/15